README.md

```diff
@@ -271,12 +289,28 @@ X-CSRFToken: required csrf token string obtained from [GET /nonce]
 
 ### `GET /api/pulse/login?original_url=<url>`
 
+**Note:** This API route does not have a version in it since it does not rely on data in the database.
+
 This will kick off a Google OAuth2 login process. This process is entirely based on browser redirects, and once this process completes the user will be redirect to `original_url` with an additional url query argument `loggedin=True` or `loggedin=False` depending on whether the login attemp succeeded or not.
 
 ### `GET /logout`
 
+**Note:** This API route does not have a version in it since it does not rely on data in the database.
+
 This will log out a user if they have an authenticated session going. Note that this route does not have a redirect path associated with it: simply calling `/logout` with an XHR or Fetch operation is enough to immediately log the user out and invalidate their session at the API server. The only acknowledgement that callers will receive around this operation succeeding is that if an HTTP 200 status code is received, logout succeeded.
 
+### `GET /api/pulse/status/`
+
+**Note:** This API route does not have a version in it since it does not rely on data in the database.
+
+This is a healthcheck route that can be used to check the status of the pulse API server. The response contains the following information:
+
+```
+{
+  latestApiVersion: <version string>
```

Comment:
if we're adding this, does it make sense to also add a `releaseCommit` that tells us which version of the codebase is running?

Reply:
Filed https://github.com/mozilla/network-pulse-api/issues/290